### PR TITLE
less relative paths for components and such thanks

### PR DESCRIPTION
**Pull request by joelhooks**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,18 +72,47 @@
...
     "postcss-preset-env": "^6.7.0",
+    "prettier": "^2.0.5",
     "react-test-renderer": "^16.13.1",
     "start-server-and-test": "^1.11.2",
     "tailwindcss": "^1.4.0",
     "typescript": "^3.9.7"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
```
Comment (joelhooks) [inline package.json:92]: very consistent formatting!

File: tsconfig.json [hunk 2]
```diff
@@ -13,7 +13,16 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "noImplicitAny": false
+    "noImplicitAny": false,
+    "baseUrl": "./",
+    "paths": {
+      "@components/*": ["src/components/*"],
```
Comment (joelhooks) [inline tsconfig.json:19]: this is a convenience method to avoid all the relative imports from `pages`

Comment (joelhooks) [reply tsconfig.json:19]: also moved the code stuff that isn't pages into `src` because the top-level directory was gonna get heinous

Comment (johnlindquist) [reply tsconfig.json:19]: Just noting that the docs use:

```js
"@/components/*": ["components/*"]
```

I'm 99% sure it's only a preference.

Comment (joelhooks) [reply tsconfig.json:19]: I was copying what Dom had done. You could make it “butts” and it would still work so it’s definitely a pref.

Comment (johnlindquist) [reply tsconfig.json:19]: +1 for "butts"

**Pull request merged**